recipe/meta.yaml

```
@@ -12,33 +12,37 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 2
```

Bumped the `build/number` again as PR ( https://github.com/conda-forge/sphinxcontrib-bibtex-feedstock/pull/26 ) was made with a feedstock branch, which resulted in `1` [being published]( https://anaconda.org/conda-forge/sphinxcontrib-bibtex/files?version=2.6.0 ). So `2` will give us a fresh build with these changes

---

Actually let's go back to `1` since the `build/number` was reset in the `2.6.1` release PR: https://github.com/conda-forge/sphinxcontrib-bibtex-feedstock/pull/28

```suggestion
  number: 1
```